client/src/login.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import "./login.css";
+
+const Login = ()=>{
+  return (
+    <React.Fragment>
+    <img src="https://user-images.githubusercontent.com/36166288/45061767-924b3300-b0ae-11e8-93fc-8e935fec045a.png" class="mainIcon" alt="icon">
+    <div>
+      <input class="username" placeholder="username"> </input>
```

Comment:
input tags are self-closing, so it would look something like this `<input />`
Next line too